xbmc/video/windows/GUIWindowVideoBase.cpp

```diff
@@ -380,6 +381,7 @@ bool CGUIWindowVideoBase::ShowIMDB(CFileItemPtr item, const ScraperPtr &info2, b
   {
     if (!info || info->Content() == CONTENT_NONE) // disable refresh button
       item->SetProperty("xxuniqueid", "xx" + movieDetails.GetUniqueID());
+    item->SetProperty("CheckAutoPlayNextItem", IsActive());
```

Comment:
`IsActive()` set from the base class will probably lead to a take 3 :)

Reply:
What do you mean?

Reply:
Just a feeling, the implementation is not easy to understand. You autoplay next item based on that property and the property is true if a given implementation of videobase is active. I guess you don't have any sort of control on the result of `IsActive` depending on whatever the derived class does

Reply:
> result of IsActive depending on whatever the derived class does

 That's nothing the derived class can influence, this is managed by the window manager. If a window gets activated by the manager, it gets state active. And there can only be one window in the system has has state active.
 
 And this is exactly what I need here. The autoplay feature only works if an instance derived from `CWindowVideoBase" is active, but not, if current active window is the Home Screen, for example.

Reply:
Ok, I got it now.